fatf/transparency/models/feature_influence.py

```diff
@@ -623,3 +660,407 @@ def partial_dependence(dataset: np.ndarray,
+        else:
+            warnings.warn(
+                'Permutation Feature Importance (PFI) requires'
+                ' scikit-learn>=0.22.0 to ben installed to allow'
+                ' user input scoring metrics.  As scikit-learn>=0.22.0 is'
+                ' not installed, PFI will run with default metrics which'
+                ' are accuracy for classifiers and max error for'
+                ' regression', UserWarning)
+
+        if as_regressor is None:
+            warnings.warn(
+                'As as_regressor was not specified,'
+                ' model will be treated as a classifier and'
+                ' accuracy will be used as scoring metric.', UserWarning)
+
+    else:
+        if ftst.is_sklearn_model(model):
+            pass
+        else:
+            if as_regressor is None:
+                if scoring_metric is None:
+                    warnings.warn(
+                        'As scoring_metric'
+                        ' and as_regressor were not'
+                        ' specified, model will be treated as'
+                        ' classifier and accuracy'
+                        ' will be used as scoring metric.', UserWarning)
+                else:
+                    pass
+            else:
+                if scoring_metric is None:
+                    if as_regressor is True:
+                        warnings.warn(
+                            'As scoring_metric was not'
+                            ' specified,'
+                            ' max_error will be used as'
+                            ' scoring metric.', UserWarning)
+                    elif as_regressor is False:
+                        warnings.warn(
+                            'As scoring_metric'
+                            ' was not specified,'
+                            ' accuracy will be used as'
+                            ' scoring metric.', UserWarning)
+
+    is_input_ok = True
+    return is_input_ok
+
+
+def get_scores(dataset: np.ndarray,
+               model: object,
+               target: np.ndarray,
+               as_regressor: bool,
+               scoring_metric: Optional[str] = None):
+    """
+    Returns the difference between the trained model's predictions
+    on the ``dataset`` and the ground truth ``target`` according to a
+    particular ``scoring_metric``. For the exceptions description please
+    see the documentation of the
+    :func`fatf.transparency.model.feature_influence.
+    permutation_feature_importance` function.
+    Parameters
+    ----------
+    dataset : numpy.ndarray
+        A dataset based on which model predictions and associated
+        predictive error will be alculated
+    model
+        A fitted model whose predictions will be used to calculate
+        associated predictive error based on 'scoring_metric'. (Please
+        see :class:`fatf.utils.models.models.Model` class documentation for the
+        expected model object specification.)
+    target: np.ndarray
```

Comment:
In the documentation, please spell out the full package name:

```suggestion
    target : numpy.ndarray
```